### PR TITLE
Cleanup and Docstrings

### PR DESCRIPTION
**Pull request by aridavidson001**
mostly formatting stuff but also a tiny thing in the pivot subsystem

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/indexer/Indexer.java [hunk 1]
```diff
@@ -11,11 +11,17 @@ public Indexer(IndexerIO io) {
     this.io = io;
   }
 
+  /**
+   * sets the indexer speed
+   *
+   * @param speed
```
Comment (Ishan1522) [inline src/main/java/frc/robot/subsystems/indexer/Indexer.java:17]: capitalize ur docstrings, include bounds of speed, and units

File: src/main/java/frc/robot/subsystems/intake/Intake.java [hunk 2]
```diff
@@ -11,11 +11,21 @@ public Intake(IntakeIO io) {
     this.io = io;
   }
 
+  /**
+   * sets the angle of the pivot in rotations
+   *
+   * @param angle desired angle in rotations
+   */
   public void setPivotAngle(double angle) {
```
Comment (Ishan1522) [inline src/main/java/frc/robot/subsystems/intake/Intake.java:19]: id add a conversion from degrees to rotations here to make it easier to tune. Also capitilize stuff

File: src/main/java/frc/robot/subsystems/intake/Intake.java [hunk 3]
```diff
@@ -11,11 +11,21 @@ public Intake(IntakeIO io) {
...
     io.setPivotPosition(angle);
     Logger.recordOutput("OTBIntake/Pivot", angle);
   }
 
+  /**
+   * sets the speed of the intake rollers
+   *
+   * @param speed sets the intake roller speed in rotations/sec
```
Comment (Ishan1522) [inline src/main/java/frc/robot/subsystems/intake/Intake.java:27]: it is a dutycycle output (unitless) from -1 to 1

File: src/main/java/frc/robot/subsystems/swerve/SwerveDrive.java [hunk 4]
```diff
@@ -144,6 +149,14 @@ private void modulesPeriodic() {
     for (var module : swerveModules) module.periodic();
   }
 
+  /**
+   * drive command
+   *
+   * @param xSpeed
+   * @param ySpeed
```
Comment (Ishan1522) [inline src/main/java/frc/robot/subsystems/swerve/SwerveDrive.java:156]: if ur gonna write a docstring, add the docs.

File: src/main/java/frc/robot/subsystems/swerve/SwerveDrive.java [hunk 5]
```diff
@@ -111,6 +111,11 @@ public void periodic() {
         timeStampIndex++) feedSingleOdometryDataToPositionEstimator(timeStampIndex);
   }
 
+  /**
+   * runs characterization based on volts
```
Comment (Ishan1522) [inline src/main/java/frc/robot/subsystems/swerve/SwerveDrive.java:115]: capitalize stuff

**Review by Ishan1522 — CHANGES_REQUESTED** [review]
very much not clean 👎

File: src/main/java/frc/robot/subsystems/intake/Intake.java [hunk 6]
```diff
@@ -11,11 +11,22 @@ public Intake(IntakeIO io) {
...
 
+  /**
+   * sets the angle of the pivot in degrees
+   *
+   * @param angle desired angle in degrees
+   */
   public void setPivotAngle(double angle) {
-    io.setPivotPosition(angle);
-    Logger.recordOutput("OTBIntake/Pivot", angle);
+    double angleRots = angle / 360;
```
Comment (Ishan1522) [inline src/main/java/frc/robot/subsystems/intake/Intake.java:20]: Just do 
`Units.rotationsToDegrees()`

**Review by Ishan1522 — CHANGES_REQUESTED** [review]

**Comment by aridavidson001** [comment]
the check format java thing works when I run it in the terminal, idk why it isn't working here @Ishan1522

**Comment by Ishan1522** [comment]
i fixed it @aridavidson001

**Pull request merged**